packages/core/integration-tests/test/schema-jsonld.js

```diff
@@ -1,26 +1,88 @@
-import {bundle, assertBundleTree} from '@parcel/test-utils';
+import {bundle, assertBundles, distDir, outputFS} from '@parcel/test-utils';
+import assert from 'assert';
+import path from 'path';
 
-describe.skip('schema ld+json', function() {
+function getPathToFile(relativePathToFile) {
```

Comment:
These functions are used only once, so you can inline them.

Reply:
addressed in 7f508f52750e83eba91e1f8647dbf6b94031789f